backend/python/app/services/implementations/story_service.py

```diff
@@ -679,6 +679,78 @@ def approve_all_story_translation_content(self, story_translation_id):
             )
             raise error
 
+    def finish_grading_story_translation(
+        self, reviewer_id, test_result, test_feedback, story_translation_test_id
+    ):
+        try:
+
+            if "translate" not in test_result or "review" not in test_result:
```

Comment:
WUAHHHH OK LAST NIT I SWEAR , if the admin isnt approving the user for reviewing, then `{translate: 4}` should be accepted. ie, this line should be
`if "translate" not in test_result:`
might need to test to make sure line 711 onwards works too